docs/widgetbook-cloud/reviews.mdx

```diff
@@ -1,124 +1,58 @@
-1. Integrate the [widgetbook package](https://pub.dev/packages/widgetbook) version `{{ versions.widgetbook }}`.
-1. Employ the CLI from [widgetbook_cli package](https://pub.dev/packages/widgetbook_cli).
-1. Set up our [GitHub app](/widgetbook-cloud/integrating-github).
-1. Upload a build of your base branch, e.g. `main`. Using this command `widgetbook publish --api-key <API>`. See [how to upload a build](/widgetbook-cloud/hosting).
-1. Create a new branch for your changes, e.g. `feat/awesome-feature`.
-1. Add new or change existing Widgetbook use cases in your feature branch. Note: Without changed use cases, the review feature won't work, as there would be nothing to review.
-1. Optionally, [add addons and knobs](/widgetbook-cloud/reviews#how-to-set-up-addons-and-knobs) as well as [Figma design links](/widgetbook-cloud/reviews#how-to-add-a-figma-design-link).
-1. Publish a review for the feature branch with changed Widgetbook use cases using this command: `widgetbook publish --api-key <API> --base-branch main`. This will publish both a **build and a review** for the latest commit in your feature branch.
-
-<Info>
-    Most CI/CD providers expose the `BASE-BRANCH` via environment variables for pull-requests. Check our [CI/CD Setup guide](/widgetbook-cloud/setup) for more insights.
-</Info>
-
-### Example
-
-```bash
-# Install dependencies
-flutter pub get
-# !!! 
-# !!! Run the build runner
-# !!! This is mandatory even if you have committed the generated files since 
-# !!! the generator creates files that are consumed by the Widgetbook CLI
-# !!!
-flutter pub run build_runner build --delete-conflicting-outputs
-# Build the Widgetbook
-flutter build web -t lib/ui_catalogs.widgetbook.dart
-# Install the CLI
-dart pub global activate widgetbook_cli
-# Run the CLI
-widgetbook publish --api-key API-KEY --base-branch main
-```
-
-## Review Information
-To populate a review, information regarding the changes in use cases is mandatory. When the `--base-branch` argument is used, the CLI fetches details about the altered use cases.
-
-<Info>
-Widgetbook Cloud showcases Reviews for all pull requests. But, only those Reviews with use case alterations are accessible.
-</Info>
 <Warning>
-At present, Widgetbook Cloud exclusively displays use cases that underwent changes, additions, or removals within the feature branch compared to the base branch. Consequently, existing use cases, even if unreviewed, won't be displayed.
+  Widgetbook Cloud Review currently only works with GitHub, we are planning to
+  add more Git providers in the future.
 </Warning>
 
+Widgetbook Cloud introduces a _Review_ feature, a zero-configuration UI regression detection tool that you can get out-of-the-box if you have Widgetbook setup for your project.
+
+| GitHub PRs                                                       | Widgetbook Cloud Reviews                                             |
```

Comment:
I really like this GitHub comparison! However, it's not clear immediately that you're comparing GitHub and Widgetbook and are not describing our GitHub integration.

So, I'd start with a sentence, like "Widgetbook Cloud Reviews work similarly like GitHub PRs. Instead of only comparing code changes, with Widgetbook Cloud Reviews, you can now also compare visual differences."